Project.swift

```diff
@@ -1,7 +1,7 @@
 import ProjectDescription
 import ProjectDescriptionHelpers
 
-let baseSettings: SettingsDictionary = ["EXCLUDED_ARCHS": "arm64"]
+let baseSettings: SettingsDictionary = [:]
```

Comment:
Will the meta tuist project be valid on Intel without this? I wonder if the CI failure is linked to this or not.

Reply:
I don't see why not, shouldn't Xcode select the correct architecture? From what I understand this particular setting shouldn't be used unless you explicitly cannot build a certain framework for `arm64` but Tuist should support that architecture. Let me know if I'm missing something! 

I'll look into the test, I believe there's a few more things required to get this working. From what I've seen locally we may need to build `ProjectDescrption` and automation as xcframeworks that support both archs as well as the dependencies fetched from `Dependencies.swift`. Do you know how this is currently done?

What I've noticed is on my M1 tuist fails to build/test the project because dependencies are missing architectures. You can replicate this by using normal M1 Xcode when resolving dependencies and then switching Xcode to be run via Rosetta.

Reply:
> From what I've seen locally we may need to build ProjectDescrption and automation as xcframeworks that support both archs as well as the dependencies fetched from Dependencies.swift. Do you know how this is currently done?

Those frameworks should already support both archs - although they are not xcframeworks but fat libs instead. You can see how they are built [here](https://github.com/tuist/tuist/blob/main/projects/fourier/lib/fourier/utilities/swift_package_manager.rb)

> What I've noticed is on my M1 tuist fails to build/test the project because dependencies are missing architectures. You can replicate this by using normal M1 Xcode when resolving dependencies and then switching Xcode to be run via Rosetta.

Ouch, that sucks. Is that something which could be fixed via build settings (which we can change in `Dependencies.swift`) or are some of the dependencies just not buildable on arm?

Reply:
This is within the tuist project so they should all be buildable. Do you know what could be happening @danyf90

Reply:
no idea 🤔  is it failing locally?

Reply:
Yeah if you have an M1 you can reproduce by fetching with default xcode selected. Then trying to build after switching to Rosetta xcode

Reply:
I should get M1 sometime next week, it's hard to help here without being able to reproduce the issue 😞